### PR TITLE
feat: integration talisman subwallet 

### LINKED ISSUE #158: 账户切换数据重载的面板应该显示 loading
比如 https://apps.darwinia.network/staking?active=power staking页面包含四个面板需要刷新数据，如果切换账户后数据是没有立即刷新的，应该新增 loading 状态来确保用户会等待新的数据拉取完成，而不是误认为页面的数据是当前账户的

### LINKED ISSUE #166: Integrate Tailsman/SubWallet

### LINKED ISSUE #176: About the problem that the account available settings of polkadot{.js} are not synchronized with the account in the apps
Need further discussion

### LINKED ISSUE #177: when using some wallet plugins that Apps do not support completely yet, it will bring some possible problems
1. When opening Darwinia Apps, other plugins will be activated, and the account address created by all other wallet plugins will be added to the account selector. It is not certain whether the selected wallet plugin can be signed correctly.  
2. When claiming staking rewards, another wallet plugin (non-Polkadot{.js} wallet)  will be called to sign, which may not be fully compatible with the current tx queue.

### LINKED ISSUE #197: Remove tx provider

### LINKED ISSUE #206: Staking assets overview add staking data loading

### PR DESCRIPTION
**Pull request by JayJay1024**

### CODE REVIEW

**Review by sxlwar — DISMISSED** [review]

**Comment by sxlwar** [comment]
@JayJay1024  conflict

**Review by ZolaxZola — DISMISSED** [review]

File: src/providers/account.tsx [hunk 1]
```diff
@@ -1,59 +1,87 @@
...
+  const { accounts } = useWallet();
+  const [account, setAccount] = useState<Account | null>();
+  const { assets, loading: assetsLoading, getAssets: refreshAssets } = useAssets(account?.displayAddress || '');
+  const [visible, setVisible] = useState(false);
+
+  const selectAccount = useCallback(
+    (address: string) => {
+      setAccount(accounts.find((acc) => acc.address === address));
+    },
+    [accounts]
```
Comment (sxlwar) [inline src/providers/account.tsx:30]: Did the editor report no type error of setAccount?

Comment (JayJay1024) [reply src/providers/account.tsx:30]: no, why would it report error

File: src/providers/wallet.tsx [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+      });
+    });
+
+    return () => sub$$();
+  }, [walletToUse, network.ss58Prefix, api, accountsObsData]);
+
+  useEffect(() => {
+    const injecteds = window.injectedWeb3;
+
+    setSupportedWallets([
```
Comment (sxlwar) [inline src/providers/wallet.tsx]: The line from 154 to 186 seems to be an configuration， isolate it?

File: public/locales/zh/translation-apps.json [hunk 3]
```diff
@@ -136,6 +136,7 @@
   "Stash account (increase the amount at stake)": "资金账户 (收益直接质押)",
   "Stop Nominating": "停止提名",
   "Stop Validating": "停止验证",
```
Comment (sxlwar) [inline public/locales/zh/translation-apps.json:138]: Untranslate

**Review by sxlwar — CHANGES_REQUESTED** [review]

File: src/providers/account.tsx (hunk 1 above)
Comment (sxlwar) [reply src/providers/account.tsx:30]: accounts.find((acc) => acc.address === address) :  Account | undefined. declared type：Account | null

Comment (JayJay1024) [reply src/providers/account.tsx:30]: declared type：Account | null | undefine

image

did you report an error there

File: public/locales/zh/translation-apps.json (hunk 3 above)
Comment (JayJay1024) [reply public/locales/zh/translation-apps.json:138]: resolved

File: src/providers/wallet.tsx (hunk 2 above)
Comment (JayJay1024) [reply src/providers/wallet.tsx]: isolated

**Pull request merged**